app/services/api/to_measurements_array.rb

```diff
@@ -24,11 +24,11 @@ def page
     start_time = Time.at(form.to_h[:start_time] / 1000)
     end_time = Time.at(form.to_h[:end_time] / 1000)
 
-    Measurement.with_streams(form.to_h[:stream_id]).where(time: start_time..end_time)
+    Measurement.with_streams(form.to_h[:stream_ids].split(",")).where(time: start_time..end_time)
```

Comment:
`with_streams` works with either array or single id?

Reply:
Yes, active record `where(stream_id: stream_ids)` works like that.